public/styles/components/nav-primary/_nav-primary.scss

```diff
@@ -43,17 +43,23 @@
 
   &:not(.is-constrained) {
     /* stylelint-disable scss/selector-nest-combinators */
-    .bsds-nav-utility .bsds-nav,
     .bsds-nav-primary .bsds-nav-bar {
       /* stylelint-enable scss/selector-nest-combinators */
       padding: 0 space(md);
     }
+
+    /* stylelint-disable scss/selector-nest-combinators */
+    .bsds-nav-utility .bsds-nav-utility-container .bsds-nav {
```

Comment:
let's remove linter-disable for L 47 and L 52 now that we split them up so we can make the high specificity more visible here.